Expensify/App
# Fix bold font on android

The bold font was not being shown on Android

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/142151

### Tests
1. Run the app on Android and write a comment that is bolded. Use a word that has a `g` as that letter is very specific for this font. 
1. Also write a comment that is not bolded, best to use the exact same phrase that you did before.
1. Verify that the first comment shows bolded and the correct font, compare it with the normal comment, and analyse each letter to make sure they look similar, but one is bolded and the other is not.
1. Check iOS and Web to see if they still show bold font correctly as well.
